### ISSUE TITLE
Error during ant build-db on bio/test-all/dbmodel

### ISSUE THREAD

**Issue by gcornut** [intermine/intermine#948]
Hello,

I'm on InterMine 1.4.2 and I've encountered an error when running `ant clean build-db` in the `bio/test-all/dbmodel` folder.
The log finishes with this:
```
[model-merger] merging model additions from: /home/fedora007/git/intermine/bio/sources/fly/bdgp-insitu/bdgp-insitu_additions.xml
     [echo] Merging bio/sources/biogrid/biogrid_additions.xml...
[model-merger] merging model additions from: /home/fedora007/git/intermine/bio/sources/biogrid/biogrid_additions.xml
     [echo] Merging bio/sources/biopax/biopax_additions.xml...

BUILD FAILED
/home/fedora007/git/intermine/bio/test-all/dbmodel/build.xml:47: The following error occurred while executing this line:
/home/fedora007/git/intermine/imbuild/objectstore.xml:166: Failed to parse model from input files
```

So naturally I searched the `biogrid_additions.xml` file which was not in the indicated directory but in `bio/sources/retired/biogrid/biogrid_additions.xml`.
This error disappeared when I corrected this path in the `bio/test-all/dbmodel/build.xml` file.

Guillaume.

**Comment by julie-sullivan** [comment]
This has been fixed by 0988f1728813846c1f00b587857d7b6d273e1914, I'll push this fix into master ASAP!

Thanks!!

**Comment by julie-sullivan** [comment]
Running `build-db` in `bio/test-all/dbmodel` is now working as expected.
